tests/features/login.feature

```diff
@@ -0,0 +1,15 @@
+Feature:Login
+    As a user
+    I want to login to the application
+    So that i can access to the home page
+
+ Scenario: Login with valid credential
+     Given the user has navigated to the login page
+     When the user login with email "admin@admin.com" and password "admin123456"
+     Then the user should we navigated to the homepage.
+
+Scenario:Login with invalid credential
+    Given the user has navigated to the login page
```

Comment:
Changes suggested above changes can be applied in these steps as well.

Reply:
I think there should be we